src/presentation/verifier_controller.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { describe, expect } from '@jest/globals';
+import fastify from 'fastify';
+import { verifierController } from '.';
+import * as xmlFiles from '../test_files/xml_example_files';
+import * as fs from 'fs';
+import { IQueryResult } from '../infrastructure/i_query_result';
+
+describe('/verifier/verifyta', () => {
```

Comment:
Der er ingen test som tjekker at resultatet indeholder de korrekte bindings. At skrive ``as xxx`` tjekker ikke for at typen er overholdt for response payload.